packages/orchestrator-ui-components/src/components/WFOTimeline/styles.ts

```diff
@@ -0,0 +1,154 @@
+        });
+
+    const getStepLineStyle = (
+        timelinePosition: TimelinePosition,
+        isFirstStep: boolean,
+        isLastStep: boolean,
+    ) => {
+        const getBeforeLineColor = (timelinePosition: TimelinePosition) => {
+            switch (timelinePosition) {
+                case TimelinePosition.PAST:
+                case TimelinePosition.CURRENT:
+                    return theme.colors.primary;
+                case TimelinePosition.FUTURE:
+                default:
+                    return theme.colors.lightShade;
+            }
+        };
+
+        const getAfterLineColor = (timelinePosition: TimelinePosition) => {
+            switch (timelinePosition) {
+                case TimelinePosition.PAST:
+                    return theme.colors.primary;
+                case TimelinePosition.CURRENT:
+                case TimelinePosition.FUTURE:
+                default:
+                    return theme.colors.lightShade;
+            }
+        };
+
+        return css([
+            !isFirstStep && {
+                '::before': {
+                    ...getHorizontalLineStyle(
+                        getBeforeLineColor(timelinePosition),
+                    ),
+                },
+            },
+            !isLastStep && {
+                '::after': {
+                    ...getHorizontalLineStyle(
+                        getAfterLineColor(timelinePosition),
+                    ),
+                },
+            },
+        ]);
+    };
+
+    const getStepOuterCircleStyle = (hasContent: boolean) =>
+        css({
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            borderRadius: '50%',
+            backgroundColor: theme.colors.emptyShade,
+            height: hasContent
+                ? stepWithValueOuterDiameter
+                : emptyStepOuterDiameter,
+            width: hasContent
+                ? stepWithValueOuterDiameter
+                : emptyStepOuterDiameter,
+        });
+
+    const getStepInnerCircleStyle = (
+        stepStatus: StepStatus,
+        hasContent: boolean,
+    ) => {
+        const diameter = hasContent
+            ? stepWithValueInnerDiameter
+            : emptyStepInnerDiameter;
+
+        return css({
+            borderRadius: '50%',
+            backgroundColor: getColorForStepStatus(stepStatus),
+            height: diameter,
+            width: diameter,
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            color: makeHighContrastColor(theme.colors.text)(
+                getColorForStepStatus(stepStatus),
+            ),
+        });
+    };
+
+    const stepStyle = css({
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+    });
+
+    return {
+        timelinePanelStyle,
+        stepStyle,
+        getStepLineStyle,
+        stepOuterCircleStyle: getStepOuterCircleStyle,
```

Comment:
why the rename?

Reply:
It used to be plain style before , but now its a function. Refactor in the IDE left the previous name to maintain the interface for the consuming components. -- Not needed, fixed it now